Contributing.md

```diff
@@ -41,22 +41,24 @@ Welcome to the CARMAPlatform contributing guide. There are a few guidelines that
 
 #### Create a New Branch
 **The number one rule is to put each piece of work on its own branch.**
-* Create a branch specific to the issue you are working on. Read the article ["Creating and deleting branches"](https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/) on GitHub. 
-* For enhancements, name the branch according to the feature e.g. **`feature/auto-activate`.**
-* For un-reported bug fixes, add a **`fix-`** prefix e.g. **`fix-admin-notices`.**
-* For code that addresses an existing Issue, add the Issue number as a prefix e.g. **`feature/123-auto-activate`** or **`feature/321-fix-admin-notices`.**
+* Check out the branch you want to make modifications to (normally this is the **`develop branch`**).
+* Create a branch specific to the issue being addressed. Read the article ["Creating and deleting branches"](https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/) on GitHub. 
+* For enhancements, name the branch according to the feature e.g. **`feature/123-auto-activate`.**
+* For un-reported bug fixes, add a **`fix-`** prefix e.g. **`fix/321-admin-notices`.**

```

Comment:
Let's avoid having changes to un-reported problems by asking them to write an issue first, then using that issue number in the branch name like the example shows.